designer/client/plugins/logger.ts

```diff
@@ -2,4 +2,36 @@ import pino from "pino";
 
 export default pino({
   name: "designer",
+  browser: {
+    transmit: {
+      level: "warn",
+      send: async function (level, logEvent) {
+        //What levels do we send to the new endpoint?
+        //Does the endpoint url matter? eg. /transmit
+        //Should this be async / do we care about the response or do we just fire and forget?
```

Comment:
> What levels do we send to the new endpoint?
We should make it configurable by env environment - for dev environments you usually want more logging (so a lower level.. or if a dev finds it annoying would change it to something higher). For higher environments, you'd want less logging, because they're actually getting stored somewhere and might start running up your aws bill! 

> Does the endpoint url matter? eg. /transmit
not really. you can have a look for common/standards but I don't think there are any

> Should this be async / do we care about the response or do we just fire and forget?
I think the server should still send a response, with status code 204 (empty response) / 5xx if it failed(?), but the designer app itself should just fire and forget. Unless you want to retry sending them, but there's not much else the designer can do 

> Do we still need browser: {asObject: true}? If so, why?
I'd recommend it. When you do `pino.error(['MyClass', 'methodThatCausedIt'], error)`, the log will be correctly parsed. Otherwise you might have to do some additional parsing on the /transmit endpoint? could be wrong!

Reply:
As discussed, we've now added an environment variable called "REACT_LOG_LEVEL" which will dictate the log level, else fall back on the NODE_ENV to determine the log level. This should allow developers to specify their own log level for the react client.